packages/api/endpoints/distribution.js

```diff
@@ -244,7 +247,15 @@ async function handleFileRequest(req, res) {
   try {
     switch (req.method) {
     case 'GET':
-      options.ResponseCacheControl = 'private, max-age=600';
+      ({
+        'Cache-Control': options.ResponseCacheControl = 'private, max-age=600',
+        'Content-Type': options.ResponseContentType,
+        'Content-Disposition': options.ResponseContentDisposition,
+        'Content-Language': options.ResponseContentLanguage,
+        'Content-Encoding': options.ResponseContentEncoding,
+        Expires: options.ResponseExpires,
+      } = headers);
```

Comment:
Why are we limiting to these 6 headers?
The implementation here is different from TEA.  For TEA, these headers are not part of the s3 signed url.
https://github.com/asfadmin/thin-egress-app/blob/9ff6fea6d04c5fa36d1aa381715c578ee8442262/lambda/app.py#L302-L312

Reply:
Hi @jennyhliu , thanks so much for catching my mistake. I've updated the behavior from overriding response headers to adding headers to the request. There is no constraint on headers and the behavior matches TEA. I also updated the testing example to reflect this change.